api/client.go

```diff
@@ -100,6 +100,13 @@ type ServiceStatus struct {
 // UnitStatus holds status info about a unit.
 type UnitStatus struct {
 	Agent AgentStatus
```

Comment:
Is the "Agent" field the legacy agent-state value? Can we add a comment to explain what it's used for.